### PR TITLE
Register htmldev.is-a-good.dev

### PR DESCRIPTION
**Pull request by BeastYTX**
Added `htmldev.is-a-good.dev` using the [CLI](https://www.npmjs.com/package/@is-a-good-dev/cli).

### CODE REVIEW

File: sub-logs/htmldev.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "owner": {
+        "username": "BeastYTX",
+        "email": "xbeastgamerytx@gmail.com"
+    },
+
+    "target": {
+        "CNAME": {
+            "name": "htmldev",
+            "value": "25faa1b9d2-hosting.gitbook.io"
```
Comment (DEV-DIBSTER) [inline sub-logs/htmldev.json:10]: 404 value, nothing is displayed.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Comment by BeastYTX** [comment]
What??

**Comment by DEV-DIBSTER** [comment]
> 25faa1b9d2-hosting.gitbook.io

image

File: sub-logs/htmldev.json (hunk 1 above)
Comment (BeastYTX) [reply sub-logs/htmldev.json:10]: ????

**Comment by DEV-DIBSTER** [comment]
Closed for inactivity.

**Pull request closed without merging**